articles/getting-started-using-admob-with-react-native-expo/index.md

```diff
@@ -0,0 +1,484 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/getting-started-using-admob-with-react-native-expo/
+title: Getting started using Admob to React Native Expo 
+description: In this tutorial we are going to go over how to set up and configure Admob into a React Native app based on the Expo ecosystem.
+author: worawat-kaewsanmaung
+date: 2020-12-08T00:00:00-11:00
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/getting-started-using-admob-with-react-native-expo/hero.jpg
+    alt: React Admob with Expo
+---
+Many modern-day apps contain ads in them. The ads pop out if you want to use certain features in the app. Mainly in game apps, you need to watch an ad before reaching another level. Ads have been a popular way to monetize the app and make some profit out of it. Some sophisticated apps also provide a payment option to remove the ads from the interface. Here, we are going to incorporate the same logic but in a slightly different way.
+<!--more-->
+In this tutorial, we are going to implement Admob in the React Native application. Here, we are going to implement an interface to show the ads as well as an option to remove ads as well. In most cases, the users will have to pay to remove the ads. But here, we are going to make users watch video ads to hide the ads from the interface for a certain amount of time. For this, we are going to make use of the expo Admob package that provides the ad reward feature.
+
+### Requirement
+- It's necessary to complete the [previous tutorial](/engineering-education/how-to-control-dark-mode-in-react-native-using-redux/) before starting this one. The previous tutorial will help you understand the use of React Navigation. Also, the use of React Native Paper components, and the Redux mechanism better.
```

Comment:
I don't see why someone needs to complete a tutorial about React Native Navigation, Redux, and React Native Papers to learn about implementing Admob.

Reply:
I think you should add a prerequisite section which states that you won't be covering the fundamentals of React, React Native, and Redux.

Reply:
The topic of your previous article has nothing to do with Admobs. Since you are reusing the App you created from the previous article, I think you should provide the app's code as a starter code, and then refer to your previous article and say that if you'd like to learn how to build the starter code provided here, refer to my previous article.

Reply:
updated as your mention.

Reply:
I still see that it says it is necessary to complete your previous tutorial.

Reply:
I made update again. look good ?